core/sr-primitives/src/generic/checked_extrinsic.rs

```diff
@@ -63,4 +64,10 @@ where
 	fn weight(&self, len: usize) -> Weight {
 		self.function.weight(len)
 	}
+	fn priority(&self, len: usize) -> TransactionPriority {
+		self.function.priority(len)
+	}
+	fn is_block_full(&self, block_weight: Weight, len: usize) -> bool {
```

Comment:
okay this is not a great name since it is misleading. Should be something along the lines of `xt.will_cause_full_block() -> bool`